about.html

```diff
@@ -0,0 +1,124 @@
+        <div class="hero-title">CELEBRATING IRREECHAA FESTIVAL 2023</div>
+        <div class="description-text">
+          Irreechaa is usually celebrated during the weekend following the Meskel holiday. The festival is observed in many areas of Oromia, but, at the national level, it brings millions of Oromos from all over Oromia and non-Oromo visitors from other parts of Ethiopia to the shores of Hora (Lake) Arsadi and, as of 2019, to Hora Finfinee in the capital. 
+          <p>
+            This year we hope to expand our invitation to individuals and
+          organizations including world class Medias to promote this beautiful
+          culture to the rest of the world. By doing this we made the
+          celebration festival to be the source of tourist attraction. So if you
+          are interested to be part of celebration we hope you will consider
+          join us this year to make Irreecha festival a world class celebrated
+          festival.
+          </p>
+        </div>
+        <p class="program-place">
+          Please contact as per email for any further question about the 2023 Irreechaa<br />
+          <span class="email-link"><a href="#">
+            Celebration Festival irrechafestival2023@otc.org </a></span>
+        </p>
+      </section>
+      <section class="main-program">
+        <h3 class="main-program-title">
+          Celebrating Irreechaa Festival 2023 Logo
+        </h3>
+        <span class="description-text">
+          Pas year Irreechaa Celebration Festival
+Take a look at the last two Irreechaa Celebration Festival which took place in Finfinnee and Bishooftuu
+
+          <p class="description-text-2">
+            it brings millions of Oromos from all over Oromia and non-Oromo visitors from other parts of Ethiopia to the shores of Hora (Lake) Arsadi and, as of 2019, to Hora Finfinee in the capital. 
+          </p>
+        </span>
+        <button class="join-btn"> <div class="logo"><a href="index.html">
+          <img src="./assets/images/titleicon.png" alt="logo-icon" class="linklogo-image" /></a></div></button>
+      </section>
+      <hr>
+      <section class="main-program">
+        <h3 class="main-program-title">Past year Irreechaa Celebration Festival</h3>
+        <span class="description-text">
+          Take a look at the last two Irreechaa Celebration Festival which took place in Finfinnee and Bishooftuu.
+        </span>
+      </section>
+    </main>
+    <section class="history-show-image">
+       <div class="image-and-text">
+        <p class="year">2019</p>
+        <p class="year-description">Celebration At Hora Finfinnee</p>
+        <img src="./assets/images/festival/fest1.jpg" alt="group-image-1">
+       </div>  
+       <div class="image-and-text">
+        <p class="year">2020</p>
+        <p class="year-description">Celebrating At Hora Har-Sadee</p>
+        <img src="./assets/images/festival/fest2.png" alt="group-image-1">
+       </div>     
+    </section>
```

Comment:
- [ ] In the Behance design, these images are wider than the images in your design so consider fixing it.
your design
![zx](https://user-images.githubusercontent.com/96683843/226132822-19f485b2-198a-4477-a246-3ae7fca1dd0b.jpg)


Behance design
![vc](https://user-images.githubusercontent.com/96683843/226132806-c384cfac-1f41-4d72-9797-060ee2121d2d.jpg)